### PR TITLE
refactor: add a basic layout to display fetched flags

### LINKED ISSUE #8: Create button component
The button should accept `colour`, `text`, and `onClick` fn.

### PR DESCRIPTION
**Pull request by manekenpix**
Closes #8

This adds a basic layout to display flags fetched from the api.

A basic way to test this is to run the back end with

```sh
docker-compose --env-file env.production up --build
```

and once it's running, store some flags (using postman, insomnia...).

Then run the front-end (make sure to create a .env file from .env.example) and load the main page.

### CODE REVIEW

File: web/src/ambient.d.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-unused-vars */
```
Comment (chrispinkney) [inline web/src/ambient.d.ts:1]: Is this necessary?

File: web/src/lib/Selector/Selector.svelte [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    border-bottom: 1px solid var(--GRAPE_700);
+  }
+</style>
+
+<script>
+  import Button from '$lib/Button/Button.svelte';
+</script>
+
+<div>
+  <Button onClick="{() => console.log('All button')}">All</Button>
```
Comment (chrispinkney) [inline web/src/lib/Selector/Selector.svelte:13]: Keeping console logs for now?

**Review by chrispinkney — DISMISSED** [review]

File: web/src/ambient.d.ts (hunk 1 above)
Comment (manekenpix) [reply web/src/ambient.d.ts:1]: It is, otherwise we get the linter gets upset.

File: web/src/lib/Selector/Selector.svelte (hunk 2 above)
Comment (manekenpix) [reply web/src/lib/Selector/Selector.svelte:13]: Yes, I think we can keep that until we implement the functionality for those buttons.

**Review by chrispinkney — APPROVED** [review]

**Pull request merged**